packages/test-utils/src/mocks/repair/v1/handlers.ts

```diff
@@ -0,0 +1,8 @@
+import { rest } from "msw";
+import { mockRepairWorkOrders } from "./data";
+import { config } from "@mtfh/common/lib/config";
+
+export const getRepairWorkOrdersV1 = (data: any = mockRepairWorkOrders, code = 200) =>
+  rest.get(`${config.repairsHubApiUrl}/worksOrders`, (req, res, ctx) => {
+    return res(ctx.status(code), ctx.json(data));
```

Comment:
should we continue with the function approach? @cloudratha to decide

Reply:
Yeah it's proving useful
```ts
ctx.json(typeof data === "function" ? data(req) : data)
```